lnwire/custom.go

```diff
@@ -21,10 +78,12 @@ type Custom struct {
 // interface.
 var _ Message = (*Custom)(nil)
 
-// NewCustom instanties a new custom message.
+// NewCustom instantiates a new custom message.
 func NewCustom(msgType MessageType, data []byte) (*Custom, error) {
-	if msgType < CustomTypeStart {
-		return nil, errors.New("msg type not in custom range")
+	if msgType < CustomTypeStart && !IsCustomOverride(msgType) {
+		return nil, fmt.Errorf("msg type: %d not in custom range: %v "+
+			"and not overridden: %v", msgType, CustomTypeStart,
+			customTypeOverride)
```

Comment:
I think this usage of `customTypeOverride` needs the read lock

Reply:
Thought I'd removed it, I don't think it's worth printing the whole map every time so simpler without it.